src/publisher/rempl-publisher.ts

```diff
@@ -132,5 +135,61 @@ publisher.provide("resolve-source-locations", locations =>
   )
 );
 
+const HIGHLIGHTER_NS = "highlighter";
+let overlay: Overlay | null = null;
+let highlighter: Highlighter | null = null;
+
+publisher.ns(HIGHLIGHTER_NS)
+  .provide("highlight", (fiberId: number, name: string) => {
```

Comment:
Maybe `startHighlight` & `stopHighlight` as for inspect?

Reply:
Fixed